Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -104,6 +105,24 @@ public enum TypeDescription: Codable, Hashable {
       } else {
         return "\(parentType.asSource).\(name)<\(generics.map { $0.asSource }.joined(separator: ", "))>"
       }
+    case let .attributed(type, specifier: specifier, attributes: attributes):
```

Comment:
Can specifiers only exist on attributed types?

Reply:
As far as I'm aware, this is correct.

Reply:
I did a little more digging. `ClosureCaptureItemSyntax` also has a `specifier`. I'm not yet sure what the options are for this syntax.

Reply:
![Screen Shot 2021-11-05 at 6 00 29 PM](https://user-images.githubusercontent.com/1791049/140592542-583cef03-9c11-42a5-8060-6733427072b6.png)

I wonder if `weak` is a specifier?